src/System.Runtime.Serialization.Json/tests/Performance/DcjsPerformanceTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.IO;
+using Newtonsoft.Json;
+
+namespace System.Runtime.Serialization.Json.Tests.Performance
+{
+    internal class DcjsSerializerFactory : SerializerFactory
+    {
+        private static DcjsSerializerFactory _instance = null;
+        public static DcjsSerializerFactory GetInstance()
+        {
+            if (_instance == null)
```

Comment:
Lazy initialization pattern is overkill. Set _instance in a static constructor which will only get run the first time the class type is referenced. Or simply instantiate the factory and set _instance on the field declaration line as the compiler will implicitly place this in the static constructor anyway.